app/Plugin/CoreApi/Model/CoreApi.php

```diff
@@ -26,18 +26,33 @@
  */
 
 App::uses("AppModel", "Model");
+App::uses('Utility', 'Validation');
 
 class CoreApi extends AppModel {
   // Define class name for cake
   public $name = "CoreApi";
 
   // Required by COmanage Plugins
-// XXX could have an "api" plugin type though there's not much value add for it atm
+  // XXX could have an "api" plugin type though there's not much value add for it atm
   public $cmPluginType = "other";
   
   // Add behaviors
   public $actsAs = array('Changelog', 'Containable');
-  
+
+  // List of allowed query parameters
+  // 1. range, inList, comparison are all validation rules supported by CAKE core
+  //    In runtime it will be translated to Validation::validation_rule, Validation::email
+  // 2. https://www.php.net/manual/en/language.variables.external.php#language.variables.external.dot-in-names
+  //    Currently the dot character is not a valid character for a PHP variable name. So co_person.status
+  //    will be transformed to co_person_status.
+  // XXX We are making the convention that the field will be placed last and will be a single word
```

Comment:
I feel like this will be a problem eventually, but it seems our options are limited.